README.md

```diff
@@ -47,7 +47,7 @@ An example Go test file has been provided in [main_test.go]().
 You can run the test suite with:
 
 ```bash
-$ TEST_ZONE_NAME=example.com go test .
+$ TEST_ZONE_NAME=snowdrop.me. go test .
```

Comment:
I think we should keep this as `example.com.` to make it obvious that it needs to be changed to some domain that the user owns

Reply:
I agree with you